README.md

```diff
@@ -6,12 +6,31 @@ This stack is based on [this CloudFormation example.](https://us-west-2.console.
 
 The stack also creates a small EC2 instance (defined in ec2-test.tf) that will be configured with a kinesis agent to test writing into the stream. If you do not wish to deploy this instance, move this file out of the terraform directory or change the extension of the file.
 
-## Deployment
+## Usage
+
+This stack is meant to be consumed as a module in your existing terraform stack. You can consume it by using code similar to this:
+
+```hcl
+module "ekk_stack" {
+    source = "github.com/GSA/devsecops-ekk-stack//terraform"
+    s3_logging_bucket_name = "${var.s3_logging_bucket_name}"
+    es_kinesis_delivery_stream = "${var.es_kinesis_delivery_stream}"
+}
+```
+
+...where the variables referenced above are defined in your terraform.tfvars file. "var.s3_logging_bucket_name" should be set to a bucket (which the stack will create) to contain copies of the kinesis firehose logs. "var.es_kinesis_delivery_stream" should be set to the name of the firehose delivery stream that you wish to use. The EKK stack will create this delivery stream with the name you provide with this variable.
+
+The Kinesis stream will send to Elasticsearch and S3.
```

Comment:
We'll probably want to add a bit about "you'll need to install a logging agent on your instances", or "here's how to forward logs", even if we just link elsewhere. Can take care of that separately though.